### ISSUE TITLE
Gradient Propagation - non-trainable Pretrained Network

### ISSUE THREAD

**Issue by pGit1** [titu1994/Keras-Classification-Models#11]
@titu1994 ,

Is it possible to fine tune lower level weights in a network while upper layer weights are frozen? Not getting any gradient in lower layers when I compute gradient manually.

**Comment by pGit1** [comment]
Closing this. Cant find this answer anywhere.
